apps/api/prisma/seed/index.js

```diff
@@ -24,9 +24,58 @@ async function main() {
 }
 
 main().then(() => {
-  console.log('done')
+  console.log('done');
+  console.log('running fetchAPISAVE');
+   fetchAPISaveGenes();
 }).catch(e => {
   console.error(e)
 }).finally(async () => {
   await prisma.$disconnect()
 })
+
+// const { PrismaClient } = require('@prisma/client');
+// const prisma = new PrismaClient();
+
+async function fetchAPI() {
+  try {
+    const response = await fetch(
+      'https://ftp.ebi.ac.uk/pub/databases/genenames/hgnc/json/hgnc_complete_set.json'
+    );
+    if (!response.ok) {
+      throw new Error('Failed to fetch data');
+    }
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
+}
+
+async function saveGenes(data) {
+  try {
+    const symbols = data.response.docs.map((x) => x.symbol);
+    for (const symbol of symbols) {
```

Comment:
Do we need to map the symbol then do a for loop again? it could just be one forReach, or a for loop.

Reply:
Would that look closer to below? Not sure if we need to map then do a for loop again but it was working albeit maybe not the msot efficiently 
```
data.response.docs.forEach(async (x) => {
      await prisma.gene.create({
        data: {
          hugoSymbol: x.symbol
```

Reply:
Yup